pcs_test/Makefile.am

```diff
@@ -0,0 +1,386 @@
+			  tools/cib.py \
+			  tools/case_analysis.py \
+			  tools/fixture.py \
+			  tools/xml.py \
+			  tools/fixture_cib.py \
+			  tools/bin_mock/pcmk/crm_resource.d/list_ocf_providers \
+			  tools/bin_mock/pcmk/crm_resource.d/list_agents_stonith \
+			  tools/bin_mock/pcmk/crm_resource.d/list_standards \
+			  tools/bin_mock/__init__.py \
+			  tools/__init__.py \
+			  tools/pcs_runner.py \
+			  tools/check/test_misc.py \
+			  tools/check/__init__.py \
+			  resources/corosync-3nodes-qdevice.conf \
+			  resources/crm_mon.minimal.xml \
+			  resources/capabilities.xml \
+			  resources/transitions02.xml \
+			  resources/cib-empty-3.4.xml \
+			  resources/resource_agent_ocf_pacemaker_booth-site.xml \
+			  resources/cluster.conf \
+			  resources/cib-empty-3.5.xml \
+			  resources/corosync-some-node-names.conf \
+			  resources/corosync-3nodes-qdevice-heuristics.conf \
+			  resources/resource_agent_ocf_heartbeat_dummy_utf8.xml \
+			  resources/stonith_agent_fence_unfencing.xml \
+			  resources/cib-large.xml \
+			  resources/cib-empty-withnodes.xml \
+			  resources/cib-empty-3.3.xml \
+			  resources/crm_mon_rng/fence-event-2.0.rng \
+			  resources/crm_mon_rng/crm_mon-2.3.rng \
+			  resources/crm_mon_rng/crm_mon.rng \
+			  resources/resource_agent_ocf_pacemaker_remote.xml \
+			  resources/corosync-no-node-names.conf \
+			  resources/corosync.conf \
+			  resources/cib-empty-2.3-withnodes.xml \
+			  resources/cib-empty-1.2.xml \
+			  resources/cib-largefile.xml \
+			  resources/cib-empty-with3nodes.xml \
+			  resources/cib-empty-3.2.xml \
+			  resources/qdevice-certs/qdevice-cert-request.crq \
+			  resources/qdevice-certs/qnetd-cacert.crt \
+			  resources/qdevice-certs/signed-certificate.crt \
+			  resources/qdevice-certs/final-certificate.pk12 \
+			  resources/resource_agent_ocf_heartbeat_dummy_insane_action.xml \
+			  resources/cib-empty-2.8.xml \
+			  resources/cib-empty-2.9.xml \
+			  resources/fenced_metadata.xml \
+			  resources/cib-empty.xml \
+			  resources/corosync-qdevice.conf \
+			  resources/stonith_agent_fence_simple.xml \
+			  resources/stonith_agent_fence_custom_actions.xml \
+			  resources/resource_agent_ocf_heartbeat_dummy.xml \
+			  resources/corosync-3nodes.conf \
+			  resources/cib-empty-2.6.xml \
+			  resources/resource_agent_ocf_heartbeat_ipaddr2.xml \
+			  resources/cib-empty-2.5.xml \
+			  resources/cib-empty-2.0.xml \
+			  resources/cib-tags.xml \
+			  resources/cib-empty-2.5-withnodes.xml \
+			  resources/cib-empty-3.1.xml \
+			  resources/transitions01.xml \
+			  __init__.py
+
+noinst_DATA		= tier1_symlinks
+
+tier1_symlinks:
+	for i in $(AGENTS); do \
+		src=`basename $${i}` && \
+		dst=`echo $${i} | sed -e 's#__#:#g'` && \
+		$(MKDIR_P) ${builddir}/tools/bin_mock/pcmk/crm_resource.d && \
+		rm -f $${dst} && \
+		$(LN_S) $${src} $${dst}; \
+	done
+
+clean-local:
+	for i in $(AGENTS); do \
+		dst=`echo $${i} | sed -e 's#__#:#g'` && \
+		rm -f $${dst}; \
+	done
```

Comment:
We can avoid this by changing the script a little bit. Here I made that change: https://github.com/ondrejmular/pcs/commit/a33c2a5ae7b0b4bdd077f80ed6dcdc21ad94f9a2

Reply:
Feel free to push those changes directly in the branch, I saw the commit and it´s fine with me of course. The main problem is that autotools in some versions have problems to deal with files that contain ":". Some older version don´t. I have tried escaping in different ways and failed miserably.
Not knowing how the code read the files, I used the hammer approach to rename and symlinks.